Cargo.toml

```diff
@@ -13,3 +13,7 @@ base64 = "0.13.0"
 urlencoding = "1.1.1"
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
+
+[dev-dependencies]
+tokio = { version = "0.2", features = ["rt-core", "macros"] }
```

Comment:
Tokio is now 1.5.0... Això és motiu de cese!!

Reply:
Yes, but the first user of this lib is going to be Limitador, and it's going to use Tokio 0.2 until actix-web upgrades :)